flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/src/main/java/org/apache/flink/tests/util/kafka/LocalStandaloneKafkaResource.java

```diff
@@ -214,7 +225,7 @@ private static void queryBrokerStatus(final Path kafkaDir, final Consumer<String
 				ZOOKEEPER_ADDRESS,
 				"get",
 				"/brokers/ids/0")
-			.setStderrProcessor(stderrProcessor)
+			.setStdoutProcessor(stderrProcessor)
```

Comment:
Does this still work on older kafka versions?

Reply:
Wouldn't it be safer to just process stdout and stderr (or redirect one of them)?

Reply:
@zentol Yes, I ran the test for all Kafka versions locally.

@AHeise We could, but the information we're looking for in the output is only in stdout. If it's not there we retry and the additional output we would get from stderr is ignored.

Reply:
Apparently it worked exactly the other way around with older version. So I'd just assume that it comes from any output and write the code accordingly.

Reply:
No, the output is very different. Before we were using the debug output that happened to be printed to stderr, even in case of no errors. Now we just use the regular, expected output of the command. This works with all the older versions as well, the code just didn't do it.